homework/CSharpOOP/Program.cs

```diff
@@ -0,0 +1,127 @@
+﻿#region Task_1
+//Console.WriteLine("Введите количество предметов, цену за единицу и скидку");
+//var numberOfItems = double.Parse(Console.ReadLine());
+//var price = double.Parse(Console.ReadLine());
+//var discount = double.Parse(Console.ReadLine());
+//var totalPrice = new Homework_1().GetDiscount(numberOfItems, price, discount);
+//Console.WriteLine($"Итоговая стоимость: {totalPrice}");
+#endregion
+#region Task_2
+using System.Diagnostics;
+
+//Console.WriteLine("Enter the product, its quantity and unit price");
+//string Item = Console.ReadLine();
+//var Number = double.Parse(Console.ReadLine());
+//var Price = double.Parse(Console.ReadLine());
+//Console.WriteLine("Enter the product, its quantity and unit price");
+//string secondItem = Console.ReadLine();
+//var secondNumber = double.Parse(Console.ReadLine());
+//var secondPrice = double.Parse(Console.ReadLine());
+//Console.WriteLine("Enter the product, its quantity and unit price");
+//string thirdItem = Console.ReadLine();
+//var thirdNumber = double.Parse(Console.ReadLine());
+//var thirdPrice = double.Parse(Console.ReadLine());
+//Console.WriteLine("Enter discount");
+//var discount = double.Parse(Console.ReadLine());
+//Console.WriteLine(new Homework_1().GetDiscountForNamedItem(Item, Price, discount));
+//Console.WriteLine(new Homework_1().GetDiscountForNamedItem(secondItem, secondPrice, discount););
+//Console.WriteLine(new Homework_1().GetDiscountForNamedItem(thirdItem, thirdPrice, discount););
+//Console.WriteLine($"Total payable: {new Homework_1().GetDiscountItem(Number, Price, secondNumber, secondPrice, thirdNumber, thirdPrice, discount)}");
+#endregion
+#region Task_3
+
+//var optionalFirst = new Homework_1().SeveralItems(Item, Price, discount);
+//var optionalSecond = new Homework_1().SeveralItems(secondItem, secondPrice, discount);
+//var optionalThird = new Homework_1().SeveralItems(thirdItem, thirdPrice, discount);
+//var requiredFirst = new Homework_1().SeveralItems(Item, Number, Price, discount);
```

Comment:
Why number here is the second param?it is optional one. 
3 first lines of this task are correct, but on 4-5 lines please check that you call the same method as you use in 1-3 lines